lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -105,6 +98,9 @@
         <% dname = section_data['section_display_name'] %>
         <li class="nav-item"><a href="" data-section="${ section_data['section_key'] }">${_(dname)}</a></li>
       % endfor
+      %if analytics_dashboard_url:
+        <li class="nav-item"><a href="${analytics_dashboard_url}" target="_blank">${_("Insights")}</a></li>
```

Comment:
I actually prefer this implementation over the former (HTML defined in the actual dashboard's HTML file, not the Python) :)